code/game/objects/buckling.dm

```diff
@@ -82,14 +82,22 @@
   * M - The mob to be buckled to src
   * force - Set to TRUE to ignore src's can_buckle and M's can_buckle_to
   * check_loc - Set to FALSE to allow buckling from adjacent turfs, or TRUE if buckling is only allowed with src and M on the same turf.
+  * addl_ride_flags - Used for riding cyborgs and humans if we need to reserve an arm or two on either the rider or the ridden mob.
   */
-/atom/movable/proc/buckle_mob(mob/living/M, force = FALSE, check_loc = TRUE)
+/atom/movable/proc/buckle_mob(mob/living/M, force = FALSE, check_loc = TRUE, addl_ride_flags = NONE)
```

Comment:
what is `addl_ride` ? additional?
increase the name length, dnt wrte ct wrds tht r hrd tw ndrstnd.
aince this proc is `buckle_mob`, a name like `buckle_mob_flags` would be just fine
after all, this is not a riding-exclusive proc

Reply:
Done